app/javascript/stylesheets/application.scss

```diff
@@ -0,0 +1,26 @@
+@import "~bootstrap/scss/bootstrap";
+
+.height-20 {
+  height: 20vh;
+}
+
+.bg-action-movies {
+  background: no-repeat center/100% url("/assets/bg_action_movies_weapon.webp");
```

Comment:
How does Asset Pipeline work?
There was definitely a great lecture on this topic.
Why won't this code work on the prod server?